lib/src/configuration/better_player_data_source.dart

```diff
@@ -76,6 +76,10 @@ class BetterPlayerDataSource {
   ///platform.
   final BetterPlayerBufferingConfiguration bufferingConfiguration;
 
+  //flag determine hide seekbar of notification
+  //fix bug ticket: https://dw-ml-nfc.atlassian.net/browse/DAF-4473
+  final bool? isHideSeekbarNotification;
```

Comment:
```suggestion
  ///Flag determine hide seekbar of notification. Currently only supported in Android platform.
  ///Fix bug ticket: https://dw-ml-nfc.atlassian.net/browse/DAF-4473
  final bool? isSeekbarOnNotificationDisable;
```

Please use document comment `///` and note this flag only for Android.